spec/javascripts/components/escape-link.spec.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-env jasmine, jquery */
+/* global EscapeLink */
+
+describe('Escape link component', function () {
```

Comment:
Are the jasmine tests being run when we call `bundle exec rake`? (thats what our CI calls to do testing)

Reply:
Hi @theseanything - thanks.  I've added a rake task to run the Javascript tests and updated the `Rakefile` to call that automatically.  So, it's now possible to just run `bundle exec rake` and the JS tests will run as part of that.